build/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # to install it.
 FROM registry.access.redhat.com/ubi8/ubi-minimal as build
 LABEL stage=build
-RUN microdnf -y install rsync make git tar findutils diffutils
+RUN microdnf -y install rsync make git tar findutils diffutils && microdnf clean all
```

Comment:
Can we make this a separate RUN command?

Reply:
I don't think that's desirable as the yum cache would then still land in the image build cache (as it's cached after each directive, i.e. each `RUN`): https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Doing this within the same RUN directive is also the common practice within OpenShift, e.g. here https://github.com/openshift/machine-config-operator/blob/master/Dockerfile.rhel7#L13